genesyscloud/resource_exporter/resource_exporter.go

```diff
@@ -304,3 +304,22 @@ func SetRegisterExporter(resources map[string]*ResourceExporter) {
 	defer resourceExporterMapMutex.Unlock()
 	resourceExporters = resources
 }
+
+var (
+	ExportAsData          []string
+	dsMutex               sync.Mutex
+	resourceNameSanitizer = NewSanitizerProvider()
+)
+
+// The GetDataSourceItems function adds resources to the ExportAsData []string and are formatted correctly
+// The ExportAsData will be checked in the genesyscloud_resource_exporter to determine resources to be exported as data source
+func GetDataSourceItems(resourceName, itemName string) {
```

Comment:
can you modify the function name here. GetDataSourceItems --> AddDataSourceItems